### DISCUSSION TITLE
Questions in Concept Derive Tables hadm_id

### DISCUSSION THREAD

**Discussion by liuxinyuan123** [MIT-LCP/mimic-code#1799] [MIMIC-IV]
When I extract the data from the concepts table, there are many cases repeated, then I find the hadm_id are not included in derived concept tables such as "first_day_lab" and "first_day_bg" and so on, I think it may be a bug or something else? 
![b08d486f-9289-4008-a101-d1fecedef6e8](https://github.com/user-attachments/assets/9cc57957-e51c-4929-9979-7e4792ed1b3d)

**Comment by yongfanbeta** [comment]
What specific cases are repeated?  can't see from the figure

> there are many cases repeated
